content/knowledge/flare_architecure.md

```diff
@@ -0,0 +1,152 @@
+
+## Flare Time Series Oracle (FTSO)
+
+The FTSO provides externally sourced data estimates to the Flare Network in a decentralized manner.
+It does so by leveraging the distributed nature of the network and its participants.
+It generates rewards for active network participants and holders of Spark (FLR) via Signal Providers and F-Asset holders.
+
+The FTSO gets external data from Signal Providers (or Oracles) in the blockchain world.
+Signal Providers exist in the off-chain to collect data, such as asset prices, and deliver it to the FTSO.
+Signal providers on the Flare Network are responsible for delivering continuous and accurate data estimates to the FTSO to represent off-chain values on the Flare Network.
+Signal providers can provide their F-Asset and FLR votes to do this, but they also can receive delegated votes from other parties to perform this service on their behalf.
+Signal providers may take fees for this service.
+Additionally, the entire process is noncustodial.
+
+The FTSO ensures the data is fair and accurate and then makes the data available to network applications.
+
+More details regarding how FTSO constructs the prices can be found in [Constructing price distributions](https://flare.xyz/ftso-a-breakdown/) section.
+
+## F-Asset System
+
+The F-Asset System is an integral application on the Flare Network.
+It facilitates the trustless issuance of asset value onto the Flare Network.
+An F-Asset is a representation of an asset from another blockchain on the Flare Network via the State Connector System, i.e.XRP = FXRP.
+Every F-Asset minted through the F-Asset System has a 2.5x collateral-backing in FLR, which is denominated in USD.
+Agents are responsible for posting and maintaining the FLR collateral.
+Additionally, they are responsible for answering redemptions, when users decide they want to have their underlying asset back.
+XRP, Litecoin, Dogecoin, and Stellar are the first four F-Assets and the addition of more is subject to FLR holder governance.
+
+F-Asset System gives the supported cryptos the capability to run smart contracts without any changes to the underlying blockchain.
+The reason to do so is that nearly 75% of the value on public blockchains cannot be currently used in a trustless manner with smart contracts.
+
+Two important parties in the F-Asset System are responsible for issuing F-Assets: Agents and Originators.
+An originator is a person or an institution that wants to convert their cryptocurrency into an F-asset, and an agent (more likely to be an institution) is a participant on the Flare network that issues the F-asset desired by the originator.
+Agents are also responsible for posting and managing the FLR collateral used to back the minting of F-Assets.
+Additionally, as F-Asset holders have the right to claim the underlying asset, the agents are responsible for honoring redemptions promptly.
```

Comment:
I think we should add a small paragraph on what happens when things go wrong as well. For instance, if an agent doesn't honour a redemption, he is slashed, and the redeemer is refunded in Spark tokens (plus a certain bonus) instead, which which he can then buy the underlying asset on the market.